src/utils/index.js

```diff
@@ -0,0 +1,4 @@
+const capitalizeFirstLetter = (text) =>
```

Comment:
Aren't we using `lodash` library for such things? 
Check it out https://lodash.com/docs/4.17.15#capitalize

Reply:
We don't use `lodash `library in our project.